### PR TITLE
docs: Gazette September, 2019

### PR DESCRIPTION
**Pull request by sidharthachatterjee**
~Work in progress~ 

- [x] Add section about RFC for localisation
- [x] Write about materialisation in bullet point 

localhost_8000_blog_2019-09-11-gazette-september_

### CODE REVIEW

File: docs/blog/2019-09-11-gazette-september/index.mdx [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+### Notable PRs
+
+It's _really_ hard to pick out favourites from all the incredible contributions you folks make so apologies in advance if I missed anything (let [us](https://twitter.com/gatsbyjs) know on Twitter). Here are my favourites from the past month:
+
+- [Jeremy Albright](https://github.com/Js-Brecht) did some really solid work and fixed an issue where Gatsby CLI would leave your terminal in an inconsistent state in [#16714](https://github.com/gatsbyjs/gatsby/pull/16714)
+- [Nicolas Goutay](https://github.com/phacks) fixed a memory leak in IntersectionObservers (with help from [atomiks](https://github.com/atomiks)) in [#17056](https://github.com/gatsbyjs/gatsby/pull/17056)
+- [Benedikt Franke](https://github.com/spawnia) brought order to some chaos by making root plural fields non nullable in [#15321](https://github.com/gatsbyjs/gatsby/pull/15321)
+- [Mateusz Burzyński](https://github.com/Andarist) cleaned up several significant parts of Gatsby by refactoring some code paths over to XState in [#17192](https://github.com/gatsbyjs/gatsby/pull/17192)
```
Comment (Andarist) [inline docs/blog/2019-09-11-gazette-september/index.mdx]: just to clarify - it has been using XState already, I've just upgraded it to XState@4, refactored a little bit to use newer APIs, so the visualization is way better than what could be achieved with older implementation

Comment (sidharthachatterjee) [reply docs/blog/2019-09-11-gazette-september/index.mdx]: Yup, thank you once again for all of that!

File: docs/blog/2019-09-11-gazette-september/index.mdx [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+Here is what [Dennis Brotzky](https://twitter.com/_brotzky) (Co-founder, [narative.co](https://www.narative.co/)) had to say:
+
+> We’ve always enjoyed using Gatsby for our projects and when themes were announcement we knew it would be special. Novela is our first dive into themes and we’ve quickly grown to love it. It’s exciting to be part of the formation of a new way to build and share Gatsby sites.
+
+## Gatsby Days, London
+
+The entire Gatsby team is headed to London in a few weeks for our quarterly Gatsby gathering!
+
+And with that, comes the next Gatsby Days. If you haven't registered already, please do [here](https://www.gatsbyjs.com/resources/gatsby-days/) to watch the Gatsby team and friends talk about exciting new things in the Gatsby world.
```
[the remark this answers is not in the document]
Comment (sidharthachatterjee) [reply docs/blog/2019-09-11-gazette-september/index.mdx]: ✅

File: docs/blog/2019-09-11-gazette-september/index.mdx [hunk 3]
```diff
@@ -0,0 +1,119 @@
+---
+title: "Gatsby Gazette: The September Installment"
+date: 2019-09-09
+author: Sidhartha Chatterjee
+excerpt: "It's September and we're back with a brand new Gatsby Gazette, this time featuring a theme of the month, an RFC to localise documentation and Gatsby Days"
```
Comment (m-allanson) [inline docs/blog/2019-09-11-gazette-september/index.mdx]: UK vs US spelling

```suggestion
excerpt: "It's September and we're back with a brand new Gatsby Gazette, this time featuring a theme of the month, an RFC to localize documentation and Gatsby Days"
```

File: docs/blog/2019-09-11-gazette-september/index.mdx [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+
+Marcy also presented a full-length workshop on [Accessibility in JavaScript Applications for Frontend Masters](https://frontendmasters.com/workshops/javascript-accessibility/), including some exclusive tidbits about building accessible Gatsby sites. In case you haven't read her post or seen her workshop already, please _do_ to learn how to make your web apps more accessible.
+
+### GraphQL
+
+We've made a bunch of improvements to Gatsby's GraphQL layer recently. Here is a quick summary:
+
+- We now support abstract data interfaces. This means that we can now build Gatsby sites (or themes) that support the same query for multiple data sources. [Mikhail](https://twitter.com/freiksenet) from the core team talks more about this on [Learn With Jason](https://www.youtube.com/watch?v=ALspNtrOqDk)
```
Comment (m-allanson) [inline docs/blog/2019-09-11-gazette-september/index.mdx]: Use 'you', and remove a word.
```suggestion
- We now support abstract data interfaces. This means that you can  build Gatsby sites (or themes) that support the same query for multiple data sources. [Mikhail](https://twitter.com/freiksenet) from the core team talks more about this on [Learn With Jason](https://www.youtube.com/watch?v=ALspNtrOqDk)
```

File: docs/blog/2019-09-11-gazette-september/index.mdx [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+- Materialisation
+- [Stefan](https://github.com/stefanprobst) added the ability to snapshot your Gatsby schema in [gatsby-plugin-schema-snapshot](https://www.gatsbyjs.org/packages/gatsby-plugin-schema-snapshot/). This enables you lock-down a project’s GraphQL schema.
```
Comment (m-allanson) [inline docs/blog/2019-09-11-gazette-september/index.mdx]: missing word
```suggestion
- [Stefan](https://github.com/stefanprobst) added the ability to snapshot your Gatsby schema in [gatsby-plugin-schema-snapshot](https://www.gatsbyjs.org/packages/gatsby-plugin-schema-snapshot/). This enables you to lock-down a project’s GraphQL schema.
```

File: docs/blog/2019-09-11-gazette-september/index.mdx [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+- GraphiQL Explorer is now available in WPGraphQL as well
```
Comment (m-allanson) [inline docs/blog/2019-09-11-gazette-september/index.mdx]: Link to PR and credit author
```suggestion
- GraphiQL Explorer is now available in the WPGraphiQL plugin for WordPress. Thanks to [Henrik](https://github.com/henrikwirth) for [adding this feature](https://github.com/wp-graphql/wp-graphiql/pull/16)
```

File: docs/blog/2019-09-11-gazette-september/index.mdx [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+
+### gatsby-plugin-offline v3
+
+[David](https://twitter.com/davidbailey00) (who recently joined the Gatsby core team) has been working on [gatsby-plugin-offline](https://www.gatsbyjs.org/packages/gatsby-plugin-offline/) for a while and recently released v3 of the plugin.
```
Comment (m-allanson) [inline docs/blog/2019-09-11-gazette-september/index.mdx]: unneeded words? feel free to reject this :)
```suggestion
[David](https://twitter.com/davidbailey00) (who recently joined the Gatsby core team) has been working on [gatsby-plugin-offline](https://www.gatsbyjs.org/packages/gatsby-plugin-offline/) and recently released v3 of the plugin.
```

File: docs/blog/2019-09-11-gazette-september/index.mdx [hunk 8]
```diff
@@ -0,0 +1,119 @@
...
+
+[Ward](https://twitter.com/wardpeet) from the core team recently made some improvements to Gatsby's webpack config in [#17093](https://github.com/gatsbyjs/gatsby/pull/17093). This introduces a better splitChunks config and results in a 400 KB reduction in initial bundle size and a 400 ms faster [TTI](https://developers.google.com/web/tools/lighthouse/audits/time-to-interactive) for [gatsbyjs.org](https://gatsbyjs.org).
+
+Huge shoutout to the Next.js team since we picked this up from them!
+
+### Renovate
+
+We recently enabled automatic dependency updates using [Renovate](https://renovatebot.com/). As of today, dependencies in all official Gatsby packages are completely up to date.
+
+This mean faster resolution of bugs caused by upstream issues and also better security.
```
Comment (m-allanson) [inline docs/blog/2019-09-11-gazette-september/index.mdx]:
```suggestion
This means faster resolution of bugs caused by upstream issues and also better security.
```

File: docs/blog/2019-09-11-gazette-september/index.mdx [hunk 9]
```diff
@@ -0,0 +1,119 @@
...
+
+### Optimized JavaScript bundle configuration
+
+[Ward](https://twitter.com/wardpeet) from the core team recently made some improvements to Gatsby's webpack config in [#17093](https://github.com/gatsbyjs/gatsby/pull/17093). This introduces a better splitChunks config and results in a 400 KB reduction in initial bundle size and a 400 ms faster [TTI](https://developers.google.com/web/tools/lighthouse/audits/time-to-interactive) for [gatsbyjs.org](https://gatsbyjs.org).
+
+Huge shoutout to the Next.js team since we picked this up from them!
+
+### Renovate
+
+We recently enabled automatic dependency updates using [Renovate](https://renovatebot.com/). As of today, dependencies in all official Gatsby packages are completely up to date.
```
Comment (m-allanson) [inline docs/blog/2019-09-11-gazette-september/index.mdx]: Including major versions?

File: docs/blog/2019-09-11-gazette-september/index.mdx [hunk 10]
```diff
@@ -0,0 +1,119 @@
...
+
+## Theme of the Month
+
+The Gatsby theme of the month is [Novela (a theme by Narative)](https://github.com/narative/gatsby-theme-novela)
+
+![Novela (a Gatsby theme by Narative)](./gatsby-theme-novela-hero.jpg)
+
+With minimal styling and maximum features — including multiple homepage layouts, built-in social sharing and dark mode — Novela makes it easy to start publishing beautiful articles and stories with Gatsby.
+
+I'm a big fan of the team at Narative and think Novela shows off some of the best part of Gatsby themes.
```
Comment (m-allanson) [inline docs/blog/2019-09-11-gazette-september/index.mdx]:
```suggestion
I'm a big fan of the team at Narative and think Novela shows off some of the best parts of Gatsby themes.
```

File: docs/blog/2019-09-11-gazette-september/index.mdx [hunk 11]
```diff
@@ -0,0 +1,119 @@
...
+
+Here is what [Dennis Brotzky](https://twitter.com/_brotzky) (Co-founder, [narative.co](https://www.narative.co/)) had to say:
+
+> We’ve always enjoyed using Gatsby for our projects and when themes were announcement we knew it would be special. Novela is our first dive into themes and we’ve quickly grown to love it. It’s exciting to be part of the formation of a new way to build and share Gatsby sites.
+
+## Gatsby Days, London
+
+The entire Gatsby team is headed to London in a few weeks for our quarterly Gatsby gathering!
+
+And with that, comes the next Gatsby Days. If you haven't registered already, please do [here](https://www.gatsbyjs.com/resources/gatsby-days/) to watch the Gatsby team (and friends) talk about exciting new things in the Gatsby world.
```
Comment (m-allanson) [inline docs/blog/2019-09-11-gazette-september/index.mdx]: avoid non-descriptive link text
```suggestion
And with that, comes the next Gatsby Days. If you haven't already, [you can register here](https://www.gatsbyjs.com/resources/gatsby-days/) to watch the Gatsby team (and friends) talk about exciting new things in the Gatsby world.
```

File: docs/blog/2019-09-11-gazette-september/index.mdx [hunk 12]
```diff
@@ -0,0 +1,119 @@
...
+
+Marcy also presented a full-length workshop on [Accessibility in JavaScript Applications for Frontend Masters](https://frontendmasters.com/workshops/javascript-accessibility/), including some exclusive tidbits about building accessible Gatsby sites. In case you haven't read her post or seen her workshop already, please _do_ to learn how to make your web apps more accessible.
+
+### GraphQL
+
+We've made a bunch of improvements to Gatsby's GraphQL layer recently. Here is a quick summary:
+
+- We now support abstract data interfaces. This means that we can now build Gatsby sites (or themes) that support the same query for multiple data sources. [Mikhail](https://twitter.com/freiksenet) from the core team talks more about this on [Learn With Jason](https://www.youtube.com/watch?v=ALspNtrOqDk)
+- Materialisation
```
Comment (m-allanson) [inline docs/blog/2019-09-11-gazette-september/index.mdx]: Something along these lines might work here?

_This work included a major refactoring in Gatsby’s data layer. It restores support for a database storage backend within Gatsby, which speeds up builds for larger sites._

File: docs/blog/2019-09-11-gazette-september/index.mdx (hunk 9 above)
Comment (sidharthachatterjee) [reply docs/blog/2019-09-11-gazette-september/index.mdx]: Good point. Minor and patch only. Will mention that in parentheses.

File: docs/blog/2019-09-11-gazette-september/index.mdx (hunk 12 above)
Comment (sidharthachatterjee) [reply docs/blog/2019-09-11-gazette-september/index.mdx]: Super useful! Thank you. I was wondering how to describe @freiksenet's incredible work here succinctly

File: docs/blog/2019-09-11-gazette-september/index.mdx [hunk 13]
```diff
@@ -0,0 +1,119 @@
...
+
+This means faster resolution of bugs caused by upstream issues and also better security.
+
+We'd like to thank [Rhys Arkins](https://twitter.com/rarkins) for all his help on setting this up and we highly recommend Renovate!
+
+## Community Highlights
+
+It's been a pretty busy month on the monorepo. We've merged in a _lot_ of your contributions and they've all been amazing. Let's go over some of the highlights!
+
+### GraphiQL Code Explorer
```
Comment (lannonbr) [inline docs/blog/2019-09-11-gazette-september/index.mdx]: This section should be Code Exporter

**Pull request merged**